common/pb/admin_commands.proto

```diff
@@ -26,3 +27,13 @@ message Command_ReloadConfig {
 		optional Command_ReloadConfig ext = 1002;
 	}
 }
+
+message Command_AdjustMod {
+	extend AdminCommand {
+		optional Command_AdjustMod ext = 1003;
+	}
+	optional string admin_name = 1;
```

Comment:
This information was created to allow a notification system of more than just sender / server to have data passed to and from.